stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -81,9 +80,31 @@
  * the {@code stetho-sample} for more information.
  */
 public class Stetho {
+  private static final AnalyticsLogger sLocalAnalyticsLogger = new AnalyticsLogger() {
```

Comment:
Hmm, prefer that the these implementation details be moved out of such a highly visible class.  Likewise, usage of statics here is counter to the normal way of configuring the `Stetho` instance.  Prefer instead to add a method to `Stetho.InitializerBuilder` or a more focused one to `InspectorModulesProvider` and/or `DumperPluginsProvider`.